spec/libraries/duplicator_spec.rb

```diff
@@ -440,13 +440,14 @@ def create_ar_graph
 
         it 'is duplicated by default' do
           duplicator = Duplicator.new
-          duplicator.duplicate(@sar_1).save
-
-          all_records = SimpleActiveRecord.all
-
-          expect(SimpleActiveRecord.count).to eq(2)
-          expect(all_records[0].data).to eq(all_records[1].data)
-          expect(all_records[0].id).to_not eq(all_records[1].id)
+          
+          expect do
+            @dup_sar_1 = duplicator.duplicate(@sar_1)
+            @dup_sar_1.save
+          end.to change{SimpleActiveRecord.count}.by(1)
```

Comment:
Space missing to the left of {.<br>Space missing inside {.<br>Space missing inside }.